packages/hoprd/releases.json

```diff
@@ -39,6 +39,10 @@
     "version_major": "1",
     "version_minor": "88"
   },
+  "debug-small": {
+    "git_ref": "^debug-small/.*",
+    "environment_id": "debug-xdai"
```

Comment:
Why not run on goerli which is for free? On xdai we need to pay xdai for on chain txs.

Reply:
Wanted to make it as close as possible to actual prod.